suse_migration_services/units/prepare.py

```diff
@@ -163,12 +177,16 @@ def main():
             zypp_plugins_services, '/usr/lib/zypp/plugins/services'
         )
         if os.path.exists(cloud_register_metadata):
-            log.info('Bind mounting /var/lib/cloudregister')
-            Path.create('/var/lib/cloudregister')
+            log.info(
+                'Bind mounting {0} from {1}'.format(
+                    cache_cloudregister_path, cloud_register_metadata
+                )
+            )
+            Path.create(cache_cloudregister_path)
             Command.run(
                 [
                     'mount', '--bind', cloud_register_metadata,
-                    '/var/lib/cloudregister'
+                    cache_cloudregister_path
```

Comment:
While the cache directory is created by the `cloud-regionsrv-client` package we should be prepared for the directory not being there. So either create it conditionally or look for and handle the failure of the mount command.

Reply:
it gets created one line above  or are you referring to something else? 

``` 
   >>>>Path.create(cache_cloudregister_path)
            Command.run(
                [
                    'mount', '--bind', cloud_register_metadata,
                    cache_cloudregister_path
                ]
            )
```